test/e2e/types/testnet.go

```diff
@@ -23,9 +24,10 @@ type Testnet struct {
 
 // EVMChain represents a EVM chain in a omni network.
 type EVMChain struct {
-	Name     string // Chain Nam.
-	ID       uint64 // Chain ID
-	IsPublic bool
+	Name        string // Chain Nam.
+	ID          uint64 // Chain ID
+	IsPublic    bool
+	BlockPeriod time.Duration // Block period
```

Comment:
Maybe nitpicking but this comment is useless. Either explain what this period is or remove the comment. No need to repeat the field name in a comment. This wouldn't help anyone who is reading the docs.